core/domain/question_domain.py

```diff
@@ -236,3 +236,38 @@ def update_question_data(self, question_data):
             question_data: dict. A dict representing the question data.
         """
         self.question_data = question_data
+
+
+class QuestionSummary(object):
+    """Domain object for Question Summary.
+
+    Attributes:
+        question_id: str. The ID of the question.
+        question_title: str. The title of the question.
+        skills: list(str). The list of skill names associated with the
```

Comment:
Shouldn't this be skill_names? In the backend, a Skill is an object that's different from a skill name.

Reply:
updated.